src/application/components/Layouts/SettingsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { Dispatch, Fragment, MutableRefObject, SetStateAction } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+
+declare const VERSION: string;
+
+export function Modal({
+  open,
+  setOpen,
+  cancelButtonRef,
+}: {
+  open: boolean;
+  cancelButtonRef: MutableRefObject<null>;
+  setOpen: Dispatch<SetStateAction<boolean>>;
+}) {
+  return (
+    <Transition.Root show={open} as={Fragment}>
+      <Dialog
+        as="div"
+        className="relative z-10"
+        initialFocus={cancelButtonRef}
+        onClose={setOpen}
+      >
+        <Transition.Child
+          as={Fragment}
+          enter="ease-out duration-300"
+          enterFrom="opacity-0"
+          enterTo="opacity-100"
+          leave="ease-in duration-200"
+          leaveFrom="opacity-100"
+          leaveTo="opacity-0"
+        >
+          <div className="fixed inset-0 bg-grey-darker bg-opacity-75 transition-opacity" />
+        </Transition.Child>
+
+        <div className="fixed inset-0 z-10 w-screen overflow-y-auto">
+          <div className="flex min-h-full items-end justify-center p-4 text-center sm:items-center sm:p-0">
+            <Transition.Child
+              as={Fragment}
+              enter="ease-out duration-300"
+              enterFrom="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+              enterTo="opacity-100 translate-y-0 sm:scale-100"
+              leave="ease-in duration-200"
+              leaveFrom="opacity-100 translate-y-0 sm:scale-100"
+              leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95"
+            >
+              <Dialog.Panel className="relative transform overflow-hidden rounded-lg bg-white px-4 pb-4 pt-5 text-left shadow-xl transition-all sm:my-8 sm:w-full max-w-sm sm:p-6">
+                <div>
+                  <div className="mt-1">
```

Comment:
```suggestion
                  <div>
```

The modal content doesn't quite feel vertically centered and this `mt-1` class seems to be the culprit. Removing this makes the modal content feel a bit more balanced in the container.

**Before**

<img width="399" alt="Screenshot 2023-10-05 at 12 30 19 PM" src="https://github.com/apollographql/apollo-client-devtools/assets/565661/59855f83-e0a7-4f5c-8b5d-2adbf9a8bf2c">


**After**
<img width="411" alt="Screenshot 2023-10-05 at 12 27 06 PM" src="https://github.com/apollographql/apollo-client-devtools/assets/565661/e83e3844-a271-4754-97ce-6f8b8760a0c9">

Reply:
Addressed in https://github.com/apollographql/apollo-client-devtools/pull/1128/commits/41bdf86d24adcb2decbd996401584e8c7914dc99